contracts/solidity/test/mock_contracts/mock_SN_messaging.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity ^0.8.21;
+
+contract mock_IStarknetMessaging {
+    function sendMessageToL2(
+        uint256 toAddress,
+        uint256 selector,
+        uint256[] calldata payload
+    ) external payable returns (bytes32, uint256) {
+        require(msg.value > 0, "L1_MSG_FEE_MUST_BE_GREATER_THAN_0");
+        require(msg.value <= 1 ether, "MAX_L1_MSG_FEE_EXCEEDED");
+        //content of contract
+        return (0x626c756500000000000000000000000000000000000000000000000000000000, 0x0); //mock values, unused.
```

Comment:
Any reason why you selected this number?

Reply:
At the moment I was having a hard time finding a compatible 'bytes32' value. Now 0x0 seems to work fine so i have changed it

Reply:
But what are those values representing? We should try to mock those values to something that makes sense.

Reply:
This mock contract has been removed, marking conversation as resolved